pages/common/gcc.md

```diff
@@ -19,6 +19,6 @@
 
 `gcc -S {{path/to/source.c}}`
 
-- Compile source code without linking:
+- Compile source code without linking, into object file:
```

Comment:
Perhaps this better
```suggestion
- Compile source code into an object file without linking:
```

Reply:
I considered that wording as well. As long as it's clear an object file by definition is without linking (I believe)

Reply:
Feel free to commit the change @jxu if it keeps the same meaning

Reply:
@navarroaxel's wording reads better for me